src/content/sessions/2023-10-06.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Desktop-Mobile: State of the Union"
+preview: ""
+isDraft: true
+date: 2023-10-06T13:00:00.000Z
+slug: "2023/10/06"
+champion: "Edgar Gonzalez, Josua Jäger"
+zoomLink: "https://us06web.zoom.us/j/82516293053?pwd=JHDBSz1xAx1Yb9zaEfLfdKfUd7r5eS.1"
+zoomPasscode: "450534"
+issueLink: "https://github.com/amplifying-fsharp/sessions/issues/21"
+company: "FundOurselves, Voyonic Systems"
+youtubeId: ""
+---
+
+## Topic
+
+In this session we will discuss the current state of the F# ecosystem for desktop and mobile development focusing on:
```

Comment:
A bit of a late nitpick but I believe you guys are quite more prepared about this session than you are letting onto. Don't you have a demo for example? I would add a little more body about what you are going to discuss. It might convince more people to join.